Makefile

```diff
@@ -49,6 +49,8 @@ endif
 export GO111MODULE=on
 GOPATH ?= $(shell go env GOPATH)
 GOFLAGS ?= $(GOFLAGS:) -mod=vendor
+export GOBIN=$(PWD)/bin
+export PATH=$(GOBIN):$(shell printenv PATH)
```

Comment:
I would not like to play around with `PATH`. What do you think about executing the tools directly, like with `./tools/bin/mmgotool`?

Reply:
The changes to `PATH` is only internal and not visible outside the `Makefile`. Hence I believe it is safe, but I am also fine with `./bin/mmgotool` if you suggest that. 2/5.